tethys_gizmos/gizmo_options/base.py

```diff
@@ -38,7 +39,52 @@ def __init__(self, attributes={}, classes=''):
         self.attributes = attributes
         self.classes = classes
 
+    @staticmethod
+    def get_tethys_gizmo_js():
```

Comment:
The get_tethys_gizmo_xx() is for the js/css applicable to all gizmos. The get_xx() is specific to the gizmo itself.